### PR TITLE
Make logs

### PR DESCRIPTION
**Pull request by sam-github**
connected to strongloop-internal/scrum-nodeops#747
connected to strongloop-internal/scrum-nodeops#403

### CODE REVIEW

**Comment by sam-github** [comment]
The tests are hosed, I'm somehow losing refs to handles and the test is exiting.

**Comment by sam-github** [comment]
@slnode test please

**Comment by sam-github** [comment]
@slnode test please

File: index.js [hunk 1]
```diff
@@ -19,30 +19,84 @@ function start(app) {
...
+    function timeout(err, data) {
+      // If we errored, we're done.
+      if (err)
+        return callback();
+
+      // If we read data, keep reading.
+      if (data) {
+        quiet = false;
+        return process.stdout.write(data, function() {
+          read(timeout);
```
Comment (rmg) [inline index.js]: Am I reading correctly that this would repeat indefinitely as long as there is something to read?

**Comment by rmg** [comment]
The tap upgrade would be a good time to add the coverage convention used by strong-central, assuming this repo works with it.

File: index.js (hunk 1 above)
Comment (sam-github) [inline index.js]: Yes, if pm/the app logs something every second. I thought about putting some kind of arbitrary max on it, but its not clear what that max would be. What about `DEBUG=* sl-start`, and you want to see all the stderr/debug up until app stops at "listening"?

I think ctrl-C is a decent control of max if you have a busy loop, or something (which will also eventually crash your system, as start.log grows without bounds...).

I'd like to see if people hit scenarios that break the heuristic before making it more complex.

**Comment by sam-github** [comment]
@slnode test please

File: index.js (hunk 1 above)
Comment (rmg) [inline index.js]: Gotcha. Seems reasonable then, as long as people aren't afraid to kill their app by hitting ctrl+c.

**Comment by sam-github** [comment]
`tap --bail --coverage --coverage-report=cobertura --timeout=200 test/test-*.` never completes, so I don't think this repo is nyc compatible.

**Comment by sam-github** [comment]
@slnode test please

**Comment by rmg** [comment]
That's not overly surprising. Thanks for checking.

LGTM, btw.

**Comment by sam-github** [comment]
thanks

**Comment by sam-github** [comment]
@slnode test please

**Comment by sam-github** [comment]
@slnode test please

**Comment by sam-github** [comment]
@slnode test please

**Comment by sam-github** [comment]
@slnode test please

**Pull request merged**